### PR TITLE
Update fallback branch references in CI & local dev tools for 2.6.x branch

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Step 4 from https://github.com/az-digital/az_quickstart/wiki/Steps-to-create-a-minor-release

This PR is targeting the new `2.6.x` release branch.

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [x] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [ ] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by joeparsons — APPROVED** [review]

**Review by jwmoore1 — APPROVED** [review]

File: .github/workflows/security.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
           if [ $(git ls-remote --heads https://github.com/az-digital/az-quickstart-scaffolding.git $AZ_TRIMMED_REF | wc -l) = 1 ]; then
             echo "SCAFFOLD_BRANCH=${AZ_TRIMMED_REF}" >> ${GITHUB_ENV}
           else
-            echo "SCAFFOLD_BRANCH=main" >> ${GITHUB_ENV}
+            echo "SCAFFOLD_BRANCH=2.6.x" >> ${GITHUB_ENV}
```
Comment (trackleft) [inline .github/workflows/security.yml:24]: Was thinking we could use this same pattern: https://github.com/az-digital/arizona-bootstrap/blob/76ecae23fd30fd8b1aeab34473003b71c29c60b3/.github/workflows/release.yml#L112

Comment (trackleft) [reply .github/workflows/security.yml:24]: But tags might put the kibosh on that.

Comment (joeparsons) [reply .github/workflows/security.yml:24]: I think that would probably work 🤔

Comment (joeparsons) [reply .github/workflows/security.yml:24]: Oh yeah maybe not.

**Pull request merged**